cmd/policy/policy.go

```diff
@@ -163,13 +163,23 @@ func NewCommand(config *settings.Config, preRunE validator.Validator) *cobra.Com
 	}()
 
 	logs := func() *cobra.Command {
-		var after, before, outputFile, ownerID, context string
+		var after, before, outputFile, ownerID, context, decisionID string
+		var policyBundle bool
 		var request policy.DecisionQueryRequest
 
 		cmd := &cobra.Command{
```

Comment:
Args with range 0 to 1 ?

Reply:
It is configured as such:
`Args:    cobra.MaximumNArgs(1),`